yt_dlp/extractor/youtube/_video.py

```diff
@@ -3091,22 +3091,34 @@ def build_fragments(f):
             if language_code and (is_original or (is_default and not original_language)):
                 original_language = language_code
 
+            has_drm = bool(fmt.get('drmFamilies'))
+
             # FORMAT_STREAM_TYPE_OTF(otf=1) requires downloading the init fragment
             # (adding `&sq=0` to the URL) and parsing emsg box to determine the
             # number of fragment that would subsequently requested with (`&sq=N`)
-            if fmt.get('type') == 'FORMAT_STREAM_TYPE_OTF':
+            if fmt.get('type') == 'FORMAT_STREAM_TYPE_OTF' and not has_drm:
                 continue
 
+            if has_drm:
+                msg = f'Some {client_name} client https formats have been skipped as they are DRM protected. '
```

Comment:
should we do this so the warnings are `only_once` *per video*?
```suggestion
                msg = f'{video_id}:  Some {client_name} client https formats have been skipped as they are DRM protected. '
```